spec/models/ability_spec.rb

```diff
@@ -192,29 +192,39 @@
       end
     end
 
-    context 'reader, editor, owner' do
+    context 'reader' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.